ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -577,6 +578,60 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskLimbOption) {
                                            options:(ORKPredefinedTaskOption)options;
 
 /**
+ Returns a predefined task that tests the participant's reaction time and impulsiveness.
+ 
+ In a go no go task, the participant is asked to move the device sharply in any
+ direction in response to a visual cue. You can use this task to accurately assess the participant's
+ simple reaction time. Randomly they will be given a cue to not shake the device, this tests 
+ impulsiveness
+ 
+ A go no go time task finishes when the participant has completed the required
+ number of attempts successfully. For a go, an attempt is successful when the participant exerts acceleration
+ greater than `thresholdAcceleration` to the device after the stimulus has been delivered and before
+ `timeout` has elapsed. An attempt is unsuccessful if acceleration greater than
+ `thresholdAcceleration` is applied to the device before the stimulus or if this does not occur
+ before `timeout` has elapsed. If unsuccessful, the result is not reported and the participant must
+ try again to proceed with the task. For a no go, the sucess condition is reversed
+ 
+ Data collected by the task is in the form of ORKGoNoGOTimeResult objects. These
+ objects contain if it was a go or no go if it was a success, and a timestamp representing the delivery,
+ of the stimulus and an ORKFileResult, which references the motion data collected during an attempt. 
+ The researcher can use these to evaluate the response to the stimulus and calculate the reaction time.
+ 
+ @param identifier                  The task identifier to use for this task, appropriate to the
+                                    study.
+ @param intendedUseDescription      A localized string describing the intended use of the data
+                                    collected. If the value of this parameter is `nil`, the
+                                    default localized text is displayed.
+ @param maximumStimulusInterval     The maximum interval before the stimulus is delivered.
+ @param minimumStimulusInterval     The minimum interval before the stimulus is delivered.
+ @param thresholdAcceleration       The acceleration required to end a reaction time test.
```

Comment:
Should there maybe be a way to specify to use a default value for this? Or at least mention a useful value in the doc comment? It almost seems like this should be set internally to the method once a good value has been determined, though I can see the value of allowing a higher threshold to be set for participants with hand tremor or other motor control issues that aren't what the test is trying to measure.